src/accel/AlongStepFactory.hh

```diff
@@ -12,6 +12,7 @@
 #include <memory>
 #include <G4ThreeVector.hh>
 
+#include "celeritas/field/RZMapFieldInput.hh"
```

Comment:
Maybe forward declare this? That'll force the downstream app to include the file where it's used.

Reply:
Have a compilation error and keep it as it is.

Reply:
That's because you have `RZMapFieldInput` as member data of the along-step factory, which shouldn't be there since we're creating it on the fly.